src/tests/challenge-manager.test.ts

```diff
@@ -5,7 +5,7 @@ import {ChallengeManager, State, stepForIndex} from '../challenge-manager';
 import {DisputeGame} from '../dispute-game';
 import {generateWitness, WitnessProof} from '../merkle';
 
-export type Role = 'challenger' | 'proposer';
+export type Identity = 'challenger' | 'proposer';
```

Comment:
Should this belong in the challenger-agent file instead?

Reply:
That's a good call! be3b0f8